components/AsanaPreview.tsx

```diff
@@ -0,0 +1,27 @@
+import styles from './AsanaPreview.module.css';
+import Link from 'next/link';
+const AsanaPreview:React.FC =() =>{
+     return(
+        <div>
+       <div><img src="https://www.dropbox.com/s/4m64ztxkj8a4dab/boatstraightlegs.svg?raw=1" className={styles.icon}/>Navasana</div>
+       <div className = 'flex flex-col'>
+        <p>English Name</p>
+        <p>Boat</p>
+        <p>Persian Name</p>
+        <p>حرکت قایق</p>
+        <p>Difficulty</p>
+        <div className='flex'>
+            <img src='/assets/images/star.png' alt="star" className={styles.star}/>
+            <img src='/assets/images/star.png' alt="star" className={styles.star}/>
+            <img src='/assets/images/star.png' alt="star" className={styles.star}/>
+            <img src='/assets/images/star.png' alt="star" className={styles.star}/>
+            <img src='/assets/images/star.png' alt="star" className={styles.star}/>
+        </div>
+
+       </div>
+        </div>
+     )
+
```

Comment:
Each asana can have a link to a YouTube video. Please add it to your asana preview component.

> [!TIP]
> To implement this, you can [embed YouTube videos in your code](https://www.w3schools.com/html/html_youtube.asp) using `iframe`.


> [!WARNING]
> In our [sample data](https://raw.githubusercontent.com/amir78729/front-end-roadmap/main/projects/yoga-positions/sample-positions.json), we have YouTube links like this:
>   https://www.youtube.com/watch?v=0495rbXZMQg ...
> What we need is a link like this to use it in an `iframe`:
> https://www.youtube.com/embed/0495rbXZMQg
> So you need to modify the URL and then pass it to the `iframe`